source/sync/error-handling/client-resets.txt

```diff
@@ -82,121 +84,219 @@ with the client's history. The most common causes of client resets are:
-   <manually-recover-unsynced-changes>` strategy.
+1. Downloads a fresh copy of the realm from the backend.
+#. Performs a diff to figure out the set of steps required to
+   bring the original (local) copy of the realm to the same state as
+   the fresh copy of the backend.
+#. Applies that set of steps to transform the local realm into a state
+   where it can sync with the backend. 
+   
+   If there are no schema changes, the SDK attempts to recover all local 
+   changes that have not yet synced to the backend. It also applies 
+   any inserts, updates, and deletes from the backend that haven't yet 
+   synced to the client.
+
+   If there are non-breaking schema changes, or if Recovery Mode cannot 
+   recover unsynced changes, the SDK can discard local changes that have 
+   not yet synced to the backend. Then it can apply any inserts, updates, 
+   and deletes from the backend that haven't yet synced to the client.
+   To fall back to discard local changes, choose your preferred SDK's 
+   version of the ``recoverOrDiscard`` client reset mode.
+
+#. Discards the fresh copy. The app continues to use
+   the original copy of the realm with the diff applied.
+
+In a breaking schema change, or if automatic client reset fails, the client 
+reset falls back to a :ref:`manual client reset handler <manual-client-reset>` 
+that your app must define. The Realm SDKs cannot automatically perform a 
+client reset when you make a breaking schema change.
+
+.. _recover-unsynced-changes:
+
+Recover Unsynced Changes
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+When **Recovery Mode** is enabled in your Device Sync configuration - as it 
+is by default - client applications can automatically recover unsynced 
+changes. In most scenarios, the client application can detect that a 
+client reset error has occurred and start an automated process to handle 
+the client reset.
+
+After the client reset, the app can open and operate as usual. 
+
+Recovery Mode can recover unsynced data in client resets except where 
+there has been a schema change. Recovery Mode applies :ref:`client-reset-recovery-rules`
+when determining how to integrate unsynced data from the device.
+
+If your app has a :ref:`non-breaking schema change <breaking-change-quick-reference>`, 
+you can choose to fall back to :ref:`discard-unsynced-changes`. You cannot 
+recover unsynced changes, but the client can automatically perform 
+the client reset. To fall back to discard local changes, choose your 
+preferred SDK's version of the ``recoverOrDiscard`` client reset mode.
+
+.. _client-reset-recovery-rules:
+
+Client Reset Recovery Rules
+```````````````````````````
+
+In a client reset that does not involve a schema change, the Realm SDKs 
+attempt to recover unsynced changes. The SDK integrates objects created 
+locally that have did not sync before the client reset. These rules 
+determine how conflicts are resolved when both the backend and the client 
+make changes to the same object:
```

Comment:
Should this paragraph specify that deletions take priority like the bullet points mention.